ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-master.yaml

```diff
@@ -68,6 +68,14 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-upgrade-gcp-loki
+- as: e2e-rollback
+  steps:
+    cluster_profile: gcp
+    env:
+      TEST_COMMAND: run-upgrade
+      TEST_OPTIONS: abort-at=100
+      TEST_SUITE: all
```

Comment:
Also the default:

```console
$ git --no-pager grep TEST_SUITE ci-operator/step-registry/openshift/upgrade/gcp/loki/
ci-operator/step-registry/openshift/upgrade/gcp/loki/openshift-upgrade-gcp-loki-workflow.yaml:      TEST_SUITE: "all"
```

Reply:
Sigh, we need to dup the defaults: https://github.com/openshift/release/pull/12789#discussion_r510414571